recipes/physx/4.x.x/conanfile.py

```diff
@@ -328,7 +328,7 @@ def package_info(self):
         self.cpp_info.components["physxcharacterkinematic"].names["cmake_find_package"] = "PhysXCharacterKinematic"
         self.cpp_info.components["physxcharacterkinematic"].names["cmake_find_package_multi"] = "PhysXCharacterKinematic"
         self.cpp_info.components["physxcharacterkinematic"].libs = ["PhysXCharacterKinematic"]
-        self.cpp_info.components["physxcharacterkinematic"].requires = ["physxfoundation"]
+        self.cpp_info.components["physxcharacterkinematic"].requires = ["physxfoundation", "physxextensions"]
```

Comment:
```suggestion
        self.cpp_info.components["physxcharacterkinematic"].requires = ["physxfoundation", "physxcommon", "physxextensions"]
```

To properly show direct dependencies.